### ISSUE TITLE
Add optional support to fit to count data with CFR/IFR correction

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/inc2prev#14]
Allows tracking of for example severity over time.

**Comment by sbfnk** [comment]
If what you're thinking to have a time-varying (with some smoothing) scaling factor (and possibly delay, though may run into identifiability issues again)? Very much agree that this would be a great addition.

**Comment by seabbs** [comment]
yes and I don't think it will as long as the delay distribution is not time-varying or at least not time-varying on the same length scale as the scaling factor (as in the CFR work).
